No write access in .gotron directory

I got a `permission denied` error at the start of creating files in the directory `.gotron`.
Turns out the `.gotron` directory, and every subdirectory, is created with permissions of `555` instead of `755`.

EDIT
My `umask` is `0022`.

```
⟩ go run main.go
panic: : : : mkdir .gotron/assets: permission denied: mkdir .gotron/assets: permission denied: : mkdir .gotron/assets: permission denied: mkdir .gotron/assets: permission denied: : : mkdir .gotron/assets: permission denied: mkdir .gotron/assets: permission denied: : mkdir .gotron/assets: permission denied: mkdir .gotron/assets: permission denied

goroutine 1 [running]:
main.main()
        /code/go/src/github.com/nanohard/NoX/main.go:24 +0x109
exit status 2

⟩ la
total 12K
drwxr-xr-x 1 user users   60 Feb 11 23:15 ./
drwxr-xr-x 1 user users  110 Feb 11 22:19 ../
drwxr-xr-x 1 user users   98 Feb 11 23:15 .git/
-rw-r--r-- 1 user users  279 Feb 11 22:45 go.mod
-rw-r--r-- 1 user users 2.9K Feb 11 22:45 go.sum
dr-xr-xr-x 1 user users    0 Feb 11 23:15 .gotron/
-rw-r--r-- 1 user users  757 Feb 11 22:23 main.go
```

--- a/Makefile
+++ b/Makefile
@@ -56,7 +56,7 @@ release:
 	-a cmd/gotron-builder/main.go
 	
 	@GOOS=windows GOARCH=amd64 CGO_ENABLED=0 \
-	go build -o ./release/gotron-builder-amd64-win \
+	go build -o ./release/gotron-builder-amd64-win.exe \
 	-ldflags="-X main.gotronBuilderVersion=$(VERSION)" \
 	-a cmd/gotron-builder/main.go
 

--- a/gotron_methods.go
+++ b/gotron_methods.go
@@ -56,11 +56,14 @@ func (gbw *BrowserWindow) CreateAppStructure(forceInstall ...bool) (err error) {
 	}
 	defer errz.Recover(&err)
 
-	//Copy Electron Files
+	err = os.MkdirAll(gbw.AppDirectory, 0777)
+	errz.Fatal(err)
+
+	// Copy Electron Files
 	err = gbw.copyElectronApplication(_forceInstall)
 	errz.Fatal(err)
 
-	//Run npm install
+	// Run npm install
 	err = gbw.runNPM(_forceInstall)
 	errz.Fatal(err)
 
@@ -139,15 +142,15 @@ func (gbw *BrowserWindow) copyElectronApplication(forceInstall bool) (err error)
 	src, err := filepath.Abs(gbw.UIFolder)
 	errz.Fatal(err)
 	dst, err := filepath.Abs(filepath.Join(gbw.AppDirectory, "assets"))
- 	errz.Fatal(err)
- 
- 	if src != dst {
- 		err = os.RemoveAll(filepath.Join(gbw.AppDirectory, "assets"))
- 		errz.Fatal(err)
- 
- 		err = copy.Copy(gbw.UIFolder, filepath.Join(gbw.AppDirectory, "assets"))
- 		errz.Fatal(err)
- 	}
+	errz.Fatal(err)
+
+	if src != dst {
+		err = os.RemoveAll(filepath.Join(gbw.AppDirectory, "assets"))
+		errz.Fatal(err)
+
+		err = copy.Copy(gbw.UIFolder, filepath.Join(gbw.AppDirectory, "assets"))
+		errz.Fatal(err)
+	}
 
 	return nil
 }

